db/migrate/20230623081549_create_favorite_recipes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFavoriteRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :favorite_recipes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :recipe, null: false, foreign_key: true
+
+      t.timestamps
+
+      add_index :favorite_recipes, [:user_id, :recipe_id], unique: true
+    end
+  end
```

Comment:
これはとてもややこしいんですが、実はindexを書く場所はもう少し下なのです、！
favorite_chefsの方も修正をお願いします！

```suggestion
    end
    add_index :favorite_recipes, [:user_id, :recipe_id], unique: true
  end
```

Reply:
失礼しました、、、！
favorite_recipesとfavorite_chefsどちらも修正しました